bc_obps/registration/api.py

```diff
@@ -166,12 +191,18 @@ def request_access(request, payload: SelectOperatorIn):
 
     # Making a draft UserOperator instance if one doesn't exist
     user_operator, _ = UserOperator.objects.get_or_create(
-        user=user, operator=operator, role=UserOperator.Roles.ADMIN, status=UserOperator.Statuses.DRAFT
+        user=user,
+        operator=operator,
+        role=UserOperator.Roles.ADMIN,
+        status=UserOperator.Statuses.DRAFT,
     )
     return 201, {"user_operator_id": user_operator.id}
 
 
-@router.get("/select-operator/user-operator/{int:user_operator_id}", response=UserOperatorOut)
+@router.get(
```

Comment:
It looks like there was a rebase error here that changed the structure of the endpoint